### DISCUSSION TITLE
Github education benefits

### DISCUSSION THREAD

**Discussion by qinshihuangtheconqueror** [community/community#144514] [GitHub Education]
### Hi everyone,

I am approved on 11 November. But til now I can't still access to Github Education. Help me

**Comment by Ajay-singh1** [comment]
Login to https://education.github.com/pack their you can access your benefits.

**Comment by davevad93** [comment]
Hi @qinshihuangtheconqueror ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @qinshihuangtheconqueror was approved and your benefits are now available.`

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
